src/app/shared/header/header.component.less

```diff
@@ -0,0 +1,15 @@
+@primary-red-color: #F40009;
+
+.navbar {
+  width: 100vw;
+
+  & a {
```

Comment:
It is a bad practice to describe styles via tags. This way can be used, for example, if you need to grow up selector specificity